### PR TITLE
luci-app-olsr-services: migrate to js

### PR DESCRIPTION
**Pull request by andibraeu**
removes lua dependencies

/cc @pmelange @aparcar

### CODE REVIEW

File: applications/luci-app-olsr-services/root/usr/share/luci/menu.d/luci-app-olsr-services.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"freifunk/services": {
```
[the remark this answers is not in the document]
Comment (andibraeu) [reply applications/luci-app-olsr-services/root/usr/share/luci/menu.d/luci-app-olsr-services.json:2]: I think so

**Comment by aparcar** [comment]
Anything missing here?

**Comment by pmelange** [comment]
Looks good to me

**Comment by jow-** [comment]
Merged, thank you!

**Pull request merged**